.github/actions/setup/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Setup
+description: Installs Node, Enables Corepack and caches pnpm.
+
+runs:
+  using: composite
+
+  steps:
+    - name: Enable corepack
+      run: corepack enable
+      shell: bash
```

Comment:
Shouldn't this step be after node lts setup? Or it doesn't matter?

Reply:
Its a workaround related to setup-node action (https://github.com/actions/setup-node/issues/480). Nuxt uses the same setup for reference